django_loci/static/django-loci/js/loci.js

```diff
@@ -377,4 +464,90 @@ django.jQuery(function ($) {
         $indoor.show();
         $indoorRows.show();
     }
+
+    var geocode_api = window.location.origin + '/admin/django_loci/location/geocode/';
+    var reverse_api = window.location.origin + '/admin/django_loci/location/reverse-geocode/';
+    window.setTimeout(function () {
+        window.address_lat = null;
+        window.address_lng = null;
```

Comment:
in javascript, use camelCase notation for variables and function names, so `addressLng`, `receivedAlert` and so on, please change all variables accordingly